cylinderDropIBM/twp_navier_stokes_p.py

```diff
@@ -49,12 +49,17 @@
                                     dragAlpha=dragAlpha,
                                     PSTAB=0.0,
                                     nParticles=1,
-                                    particle_epsFact=3.,
+                                    particle_epsFact=1.5,
                                     particle_alpha=1e5,
                                     particle_beta=1e5,
                                     particle_penalty_constant=1e6,
-                                    particle_sdfList=[obj_sdf_Calc],
-                                    particle_velocityList=[obj_vel_Calc])
```

Comment:
Okay so I think the only difference between the model you ran for the cylinder problem and the ones I had here is that you calculate the sdf for rigid balls from the position/velocity of MBD solver, whereas I gradually had changed this to calculate the sdf for any shape via the particle_sdfList, particle_velocityList. This was needed because for the flexible plate example the sdf has to be computed that way.
Anyway, I had made some changes in the proteus source codes (unfortunately in the same commit as I did the merge with the latest proteus) to be able to run these and they are available here :
https://github.com/Milad-Rakhsha/proteus
what I recall is that the bug fixes in one of the RANS*.h files I made was actually necessary to get the sdf calculator to work properly (I think there was an out-of-bound memory access that I had fixed).

Reply:
OK, sorry for the slow response. Are you saying that in your merge with the latest proteus main branch you are able to run the flexble plate?

Reply:
sorry for the confusion, no what I meant was that I could run the flexible plate problem with the branch in the link above. However, the branch is not merged with the latest master. When I last merged it (2 years ago), I incorporated all the bug fixes with the merge commit so that is why it is pretty difficult to see what was part of the merge and what was the actual bug fixes I needed to have the sdf calculators working properly

Reply:
Would it be straightforward to try to build this old branch in a current proteus conda environment? I am asking because most of the dependencies are installed within  the conda env

Reply:
I think  it should work. Most of the the time the changes to the dependencies only cause a couple of easy proteus changes, at most. (e.g. petsc and hdf5 api changes).